app/1.0.x/plugin-development/access-the-datastore.md

```diff
@@ -50,8 +39,8 @@ local consumers_dao = kong.db.consumers
 local plugins_dao = kong.dao.plugins
 ```
 
-The `kong` global exposes the [Plugin Development Kit], and its `kong.dao` and
-`kong.db` properties are instances of the DAO and DB singletons.
+Other entities core entities as well as custom plugin entities are also
```

Comment:
`Other entities core entities as well as custom plugin entities`

This does not read well. How about `Both core entities from Kong and custom entities from plugins are also available...`?